Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10-slim-bullseye
+
+WORKDIR /app
+
+COPY . /app/
+
+RUN \
+  apt update && \
+  apt install -y --no-install-recommends gcc libcairo2-dev && \
```

Comment:
Maybe we can uninstall these ones after `pip install`? If it only needed for building.

Reply:
I don't think so. Cairo library is used for PNG tile generation, Pycairo just provides bindings for it.